.gitlab-ci.yml

```diff
@@ -52,4 +56,20 @@ publish:
         Write-Output "Encountered error during while publishing to S3. Error Message is $msg."
         Write-Output "Retrying..."
         aws s3 cp artifacts/ s3://dd-windowsfilter/builds/tracer/${CI_COMMIT_SHA} --recursive --region us-east-1 --exclude "*" --include "*.zip" --include "*.msi"
-      }
\ No newline at end of file
+      }
+
+deploy_to_reliability_env:
+  stage: deploy
+  rules:
+    - if: '$FORCE_TRIGGER == "true"'
```

Comment:
> Is the purpose of this to allow deploying an arbitrary branch to the rel-env

Nope, the purpose is to set this variable to `true` in a scheduled pipeline. That way we can trigger it every day from master. This could be triggered from any branch on our end indeed as I don't restrict it, but the deployment on the reliability restricts to the last commit from master (thus why we can trigger the job manually from any pipeline)

>  We already push every master release to the rel-env, right?

We currently push only manually to the reliability environment, when we decide to do a release.

I'll remove a few variables below to make that clearer as they are useless in our case

Reply:
> We currently push only manually to the reliability environment, when we decide to do a release

Yeah, though we _were_ pushing every master, until something broke. 

IMO we should be pushing regularly to the reliability environment, and have alerting to catch any issues, not _just_ before a release (as that's often too late).

But based on your comments, looks like that's what you're doing anyway with the schedule 🙂🎉